README.md

```diff
@@ -442,9 +463,7 @@ steps:
 ```
 
 This will create a new Container App named `github-action-container-app-<github-run-id>-<github-run-attempt>` in a new
-resource group named `<container-app-name>-rg` where the image built and pushed to ACR is named
-`mytestacr.azurecr.io/app:latest`
-
+resource group named `<container-app-name>-rg` where the image built and pushed to ACR is named `mytestacr.azurecr.io/app:latest`

```

Comment:
nit: would you mind reverting the change made to this line? (running over the 120 character limit and no new line after the text before the _Contributing_ heading)